src/staging/src/components/park/parkDates.js

```diff
@@ -66,25 +92,28 @@ export default function ParkDates({ data }) {
             dateRec.reservationStartDate,
             dateRec.reservationEndDate
           )
-          // TODO winter dates
+          const offSeasonDates = datePhrase(
+            dateRec.offSeasonStartDate,
+            dateRec.offSeasonEndDate
+          )
 
-          dates.push({
-            dateCount: saDateCount,
-            subArea: subArea.parkSubArea,
-            serviceDates: serviceDates,
-            resDates: resDates,
-          })
+          if (saDateCount === 1) {
```

Comment:
I'm not sure if there are any cases where it's an issue but I think this logic will result in a leading comma if there are two dates and the first is inactive.

Recommend building 3 arrays of active date strings and using `.join(" ,")`

Reply:
Moving the increment into the isActive condition, will avoid the case with first inactive date